src/test/java/com/google/api/codegen/testdata/discoveries/storagetransfer.v1.json.overrides

```diff
@@ -13,6 +13,7 @@
               "isArray": false,
               "isMessage": false
             },
+            "placeholder": "",
```

Comment:
Well it's really here because this field isn't populated by the `DiscoveryImporter`, and so it's entire definition is specified in the overrides. We have to include it because it's a non-nullable field and the `SampleConfig` can't be built unless it's specified.